buf.gen.yaml

```diff
@@ -1,11 +1,11 @@
-version: v1
+version: v2
+clean: true
 managed:
   enabled: true
-  go_package_prefix:
-    default: github.com/bufbuild/protovalidate/tools/internal/gen
-    except:
-      - buf.build/envoyproxy/protoc-gen-validate
```

Comment:
I have no idea why PGV is in here - it isn't a dependency of any modules in this project.